wikidatarefisland/data_model/filters.py

```diff
@@ -45,3 +45,30 @@ def get_property_id_statement_includer(included_properties):
         """
         return lambda statement: statement.get('mainsnak', {}).get('property', '') \
             in included_properties
+
+
+class ItemFilters:
+    @staticmethod
+    def get_item_class_excluder(excluded_classes):
+        """
+        Exclude items by class
+
+        Return a filter that is false for all items in the provided class
+        e.g. that have any P31 statements of that class
+        """
+        class_property = 'P31'
```

Comment:
I think this is fine for now, but we should probably create a ticket to remove this (and others, I'm at fault here too, see Q2 in the GeoValue class...) to a more scalable solution.